src/pages/Inquiry/index.tsx

```diff
@@ -0,0 +1,63 @@
+import { ReactComponent as Write } from 'assets/svg/inquiry/write.svg';
+import { ReactComponent as Move } from 'assets/svg/inquiry/myinquiry.svg';
+import usePostList from 'pages/Post/hooks/usePostList';
+import { useState } from 'react';
+import { Link } from 'react-router-dom';
+import styles from './Inquiry.module.scss';
+import Pagination from './pagination';
+import Datatable from './datatable';
+
+export default function Inquiry(): JSX.Element {
+  const [page, setPage] = useState(1);
+  const { data: postData } = usePostList(page);
+
+  return (
+    <div>
+      <div className={styles.container}>
+        <header className={styles.header}>
+          <div>
+            <h1 className={styles.header__title}>문의하기</h1>
+            <h3 className={styles['header__sub-title']}>쩝쩝박사에게 궁금한 점이 있나요?</h3>
+          </div>
+          <Link to="/Myinquiry" className={styles.header__link}>
+            <div className={styles['header__my-inquiry']}>
+              나의 문의 바로가기
+              <div className={styles.header__move}>
+                <Move />
+              </div>
+            </div>
+          </Link>
+        </header>
+        {
+          postData?.content && (
+            <Datatable data={postData.content} />
+          )
+        }
```

Comment:
데이터가 있다는 조건은 `postData`만으로 작성해도 괜찮을 것 같아요. 옵셔널이 붙으니 좀 더 고려할게 늘어나는 느낌?
아래 pagination부분도 동일하구요
```suggestion
        {
          postData && (
            <Datatable data={postData.content} />
          )
        }
```

Reply:
넵 수정했습니다!